README.md

```diff
@@ -59,6 +59,7 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
 
 - [Test Deployments](https://www.cockroachlabs.com/docs/stable/deploy-a-test-cluster.html) - Easiest way to test an insecure, multi-node CockroachDB cluster.
```

Comment:
Let's remove this Test Deployments link. The page it links to is gone, though we have a redirect in place.